src/Collections-Unordered/Dictionary.class.st

```diff
@@ -333,6 +333,21 @@ Dictionary >> at: key put: anObject [
 	^ anObject
 ]
 
+{ #category : #accessing }
+Dictionary >> at: key update: updateBlock [ 
+	"I am used to update the value at a given key, or if the key does not exist, to throw an error"
+	self at: key update: updateBlock initial: [ self errorKeyNotFound: key ].
+]
+
+{ #category : #accessing }
+Dictionary >> at: key update: updateBlock initial: initObjectOrValue [
+	| index |
```

Comment:
Maybe you should state here that the fact that you use #findElementOrNil: instead of the

```Smalltalk
self at: key put: (self at: key)
```

is because this method is optimised.

Reply:
Comments is always a good thing. I will see if I am able to get it done. Git and the transition to 9.0 has given me all kinds of problems (most likely because I do not understand fully what goes on).

Reply:
Maybe for a comment you can do it from GitHub web app directly?
But you need to be careful. :-)